lib/src/views/Calendar/CalendarHeader.tsx

```diff
@@ -127,15 +181,11 @@ CalendarHeader.displayName = 'CalendarHeader';
 CalendarHeader.propTypes = {
   leftArrowIcon: PropTypes.node,
   rightArrowIcon: PropTypes.node,
-  disablePrevMonth: PropTypes.bool,
-  disableNextMonth: PropTypes.bool,
 };
 
 CalendarHeader.defaultProps = {
```

Comment:
Have you considered to use the prop spread default values instead of default props?

Reply:
It is primarily used for typescript typings. This is just so old code, will move it. (But there is a problem with auto generating default props values)

Reply:
We might have a solution for this. The default props generation for API purpose was solve in the mui-org/material-ui repository. @eps1lon took care of it.
It's a good reminder that we need to look into how we can extract the API from TypeScript sources, and not only JavaScript ones.

Reply:
Right now I created custom script that reads jsdoc tags on props like `@default "someValue"` and primarily use it and getting rid from default props.

Reply:
Our handler for `react-docgen` and default props via default values in object rest spread can be found in https://github.com/mui-org/material-ui/blob/6f7318cdd5c1c2887860c0d56212a3e42d107082/docs/src/modules/utils/defaultPropsHandler.js#L8

Reply:
@eps1lon don't you use `react-docgen-typescript`?